src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -888,11 +896,9 @@ public override void Accept(OperationVisitor visitor)
 
     internal partial class BoundBaseReference : IInstanceReferenceExpression
     {
-        bool IInstanceReferenceExpression.IsExplicit => this.Syntax.Kind() == SyntaxKind.BaseExpression;
-
-        IParameterSymbol IParameterReferenceExpression.Parameter => (IParameterSymbol)this.ExpressionSymbol;
+        InstanceReferenceKind IInstanceReferenceExpression.InstanceReferenceKind => InstanceReferenceKind.BaseClass;
```

Comment:
Do we need this distinction between this/base or is it implied from the bound target method and whether the invocation is virtual?